src/etc/maketest.py

```diff
@@ -53,6 +53,8 @@ def convert_path_spec(name, value):
 putenv('TARGET_RPATH_DIR', os.path.abspath(sys.argv[11]))
 putenv('RUST_BUILD_STAGE', sys.argv[12])
 putenv('S', os.path.abspath(sys.argv[13]))
+putenv('RUSTFLAGS', sys.argv[15])
```

Comment:
> Was this necessary to pass along? It looks like this isn't used elsewhere from this patch?

Yeah, the `execution-engine` test needs access to LLVM's dylib. Since trying to disable that test only when LLVM is shared would also require an additional env var, might as well just fix the test.

> Also this looks like it skipped a number?

Didn't skip a number; `target_triple` from earlier in the file takes arguement 14.